tests/e2e/constants/constants.go

```diff
@@ -0,0 +1,11 @@
+package constants
+
+const (
+	RunController = "cd ${GOPATH}/src/github.com/kubeedge/kubeedge/cloud/; sudo nohup ./edgecontroller > edgecontroller.log 2>&1 &"
```

Comment:
it doesn't look good to have such commands defined as constants.
can we have a script with a function which can be executed via os.exec ?

Reply:
@pavan187 , let's create an issue for this. similar comments for device management e2e test suite from Sujith.

Reply:
> it doesn't look good to have such commands defined as constants.
> can we have a script with a function which can be executed via os.exec ?

OK sure, will change as suggested